code-challenges.test.js

```diff
@@ -15,40 +15,135 @@
 
+//6. create a function "randomColor" for a test and take in an array of strings
+//7. .shift() - it's used get rid of the first index of array 
+//8. create a new variable to store a new array after index[0] is removed
+//9. run a for loop and use Math.floor(Math.random()) to shuffle indexs around
+//10. create a new variable to store a new shuffled array
+//11. return the removeFirstIndex
+//12. run yarn jest
+//13. Got green
+
+
+    const randomColors = (arr) => {
+        arr.shift()
+        let removeFirstIndex = arr
+        for (i = 0; i < removeFirstIndex.length; i++) {
+            let shuffles = Math.floor(Math.random() * removeFirstIndex.length)
+            let newSet  = removeFirstIndex[i]
+            removeFirstIndex[i] = removeFirstIndex[shuffles]
+            removeFirstIndex[shuffles] = newSet
+        }
+        return removeFirstIndex
+    }
 
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-var nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
+// var nums1 = [3, 56, 90, -8, 0, 23, 6]
+// // Expected output: [-8, 90]
+// var nums2 = [109, 5, 9, 67, 8, 24]
+// // Expected output: [5, 109]
 
+//1. Create a function "orderNums"
+//2. Takes an array of numbers and returns an array of the minimum and maximum numbers in that order
+//3. expect the input [3, 56, 90, -8, 0, 23, 6] .toEqual [-8, 90] and 
+// expect the input [109, 5, 9, 67, 8, 24] .toEqual [5, 109] 
+//4. Run yarn jest
+//5. Got red 
 
+    describe("orderNums", () => {
+        var nums1 = [3, 56, 90, -8, 0, 23, 6]
+        var nums2 = [109, 5, 9, 67, 8, 24]
+        it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {
+        expect(orderNums(nums1)).toEqual([-8, 90]),
+        expect(orderNums(nums2)).toEqual([5, 109])
+        })
+    })
 
-// b) Create the function that makes the test pass.
 
+// b) Create the function that makes the test pass.
 
+//6. create a function "orderNums" for a test and take in an array of numbers
+//7. do a for loop to go through every index in an array
+//8. create a new variable to store a new array -- newNums = []
+//9. use Math.max to get the maximum of the number in the array and store it in a maxNum variable
+//10. use Math.min to get the minimun of the number in the array and store it in a minNum variable 
+//11. push minNum and maxNum in a new variable called newNumset
+//12. return newNums
+//13. run yarn jest
+//14. Got green
+
+    const orderNums = (arr) => {
+        for (i = 0; i < arr.length; i++) {
+        newNums = []
+        maxNum = Math.max(...arr)
+        minNum = Math.min(...arr)
+        newNumsSet = newNums.push(minNum, maxNum)
+        }
+        return newNums
+    }
```

Comment:
You have some good logic here but I think you are doing a bit more work than is needed. I don't think there is any iteration needed if you are using Math.min/max. You can get the same result by just putting your min/max logic in an array. `return [Math.max(...arr), Math.min(...arr)]`